x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -82,6 +101,30 @@ export class TelemetryEventsSender {
     }
   }
 
+  public async fetchDiagnosticAlerts(fromTimestamp: string, toTimestamp: string) {
+    const query = {
+      index: 'pete-hampton-test-index*',
+      ignore_unavailable: true,
+      size: 100,
+      body: {
+        query: {
+          range: {
+            '@timestamp': {
+              gt: fromTimestamp,
+              lte: toTimestamp,
+            },
+          },
+        },
+      },
+    };
```

Comment:
`@timestamp` is probably used just for testing, but remember to range by `event.ingested`. That one is assigned when the alerts reach Elasticsearch so it's more reliable.

Not sure if it matters too much, but we should think if we want to return the first or the last 100 alerts in case there are more than that. I have a slight preference for the most recent 100 alerts because just in case the interval gets larger than 5 minutes, the more recent ones would be more valuable I think.

Reply:
I too prefer the most recent. I will chat with you offline RE the `event.*` fields. Things don't seem to be lining up with the ransomware samples I have been receiving.